bin/dump-to-git.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var fs = require('fs');
+var process = require('process');
+var child_process = require('child_process');
+var Promise = require('bluebird');
+var NodeGit = require('nodegit');
+var LZString = require('lz-string');
+var models = require("./lib/models");
```

Comment:
This yields a "Error: Cannot find module", when the script lives in `bin/` (where it's supposed to be, judging from this pull request). I have to move the script to `/` to run it beyond this point.